source/docs/software/roborio-info/imaging-your-roborio.rst

```diff
@@ -90,7 +90,7 @@ Troubleshooting
 If you are unable to image your roboRIO, troubleshooting steps include:
 
 - Try running the roboRIO Imaging Tool as Administrator by right-clicking on the Desktop icon to launch it.
-- Try accessing the roboRIO webpage with a web-browser at ``http://172.22.11.2/`` and/or verify that the NI network adapter appears in your list of Network Adapters in the Control Panel. If not, try re-installing the NI Update Suite or try a different PC.
+- Try accessing the roboRIO webpage with a web-browser at ``https://172.22.11.2/`` and/or verify that the NI network adapter appears in your list of Network Adapters in the Control Panel. If not, try re-installing the NI Update Suite or try a different PC.
```

Comment:
This actually will not work because the rio does not have a certificate. http is correct.